fe/fe-core/src/test/java/org/apache/doris/qe/RuntimeFilterTypeHelperTest.java

```diff
@@ -50,6 +50,30 @@ public void testNormal() throws DdlException {
         runtimeFilterType = "IN,BLOOM_FILTER,MIN_MAX";
         Assert.assertEquals(new Long(7L), RuntimeFilterTypeHelper.encode(runtimeFilterType));
 
+        runtimeFilterType = "IN_OR_BLOOM_FILTER";
+        Assert.assertEquals(new Long(8L), RuntimeFilterTypeHelper.encode(runtimeFilterType));
+
+        runtimeFilterType = "IN,IN_OR_BLOOM_FILTER";
+        Assert.assertEquals(new Long(9L), RuntimeFilterTypeHelper.encode(runtimeFilterType));
```

Comment:
same as above

Reply:
Just for test, we should guide user by doc.